Update default Tab index

mustafaskyer opened the pull request:
I have many routes 
[likes, search, home, profile, settings]
and suggest making the default  route to be in the middle,
or we can add defaultTab prop into tabs property which should be the default
regards

mustafaskyer committed:
Update default Tab index

I have many routes 
[likes, search, home, profile, settings]
and suggest making the default  route to be in the middle,
or we can add defaultTab prop into tabs property which should be the default
regards

timomeh commented:
Thank you very much for your contribution!

Setting it to the middle tab seems too specific for your use case. But adding the `defaultTabIndex` prop as you suggested sounds good. The same thing was already discussed in https://github.com/timomeh/react-native-material-bottom-navigation/issues/102#issuecomment-415427696. It would be great if you could implement `defaultTabIndex`.

Bonus points if _no_ tab would be active if the `defaultTabIndex` prop isn't set. 🙂 This would finally resolve #102

mustafaskyer committed:
Added defaultTabIndex property,

Added defaultTabIndex property, if exist respectively will be the default tab, otherwise, the default is the first index of tabsdefaultTabIndex should pass as a prop to , like so
<BottomNavigation defaultTabIndex={2} ... />
regards

mustafaskyer commented:
thanks, @timomeh
commit updated :),
now expects defaultTabIndex,
regards

mustafaskyer committed:
Update TabList.js
